src/foundation/components/Label/css.ts

```diff
@@ -0,0 +1,88 @@
+import { css } from '@linaria/core';
+
+// const mobileMixin = (styles: string) => css`
+//   .sendbird--mobile-mode & {
+//     ${styles}
+//   }
+// `;
+//
+// const sendbirdLabelMobile = css`
+//   ${mobileMixin(`
+//     -webkit-user-select: none;
+//     -webkit-touch-callout: none;
+//   `)};
+// `;
```

Comment:
Should we keep these comments?

Reply:
I've left only the code for the part that needs to be implemented additionally.